wpilibc/athena/src/PIDController.cpp

```diff
@@ -591,12 +591,14 @@ void PIDController::InitTable(std::shared_ptr<ITable> subtable) {
  * @return Error for continuous inputs.
  */
 double PIDController::GetContinuousError(double error) const {
-  if (m_continuous &&
-      std::fabs(error) > (m_maximumInput - m_minimumInput) / 2) {
-    if (error > 0) {
-      return error - (m_maximumInput - m_minimumInput);
-    } else {
-      return error + (m_maximumInput - m_minimumInput);
+  if (m_continuous) {
+    double mod_error = std::fmod(error, m_inputRange);
```

Comment:
For whatever reason, I was under the impression that a `const` function also couldn't modify any of the variables within it's scope (including any arguments) - I see now that was incorrect.